releasenotes/notes/enable-windows-parallel-map-b33122a2122a95d7.yaml

```diff
@@ -0,0 +1,55 @@
+---
+features:
+  - |
+    When running on Windows parallel execution with
+    :func:`~qiskit.tools.parallel_map` (primarily with
+    :func:`~qiskit.compiler.transpile` and :func:`~qiskit.compiler.assemble`)
+    can now be enabled. To do this you can either set ``parallel = True`` in
+    a user config file, or set the ``QISKIT_PARALLEL`` environment variable to
+    ``TRUE``.
```

Comment:
Do you think it'd be beneficial to direct the user to the issue note here? (They won't necessarily appear in this order in the final release notes, right?)

Reply:
Yeah, that's a good idea I'll leave a note about there being potential issues and refer to the known issues section

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/3547/commits/44e5ebceed55a70a52ae9bfc321e72e9a4ba486e